.travis.yml

```diff
@@ -0,0 +1,29 @@
+language: python
+python:
+  - "3.6"
+os: linux
+sudo: required
+dist: trusty
+compiler: gcc
+
+env:
+  - OS_TYPE=centos OS_VERSION=7
+
+services:
+  - docker
+
+install:
+  - sudo apt-get update
+  - echo 'DOCKER_OPTS="-H tcp://127.0.0.1:2375 -H unix:///var/run/docker.sock -s devicemapper"' | sudo tee /etc/default/docker > /dev/null
+  - sudo service docker restart
+  - sleep 5
+  - sudo docker pull lsstdesc/stack-sims:w_2018_09-sims_2_6_0-v2
+  - pip install coveralls
+
+script:
+  - sudo chown -R 1000:1000 .
+  - echo ${TRAVIS_BUILD_DIR}
+  - docker run -v `pwd`:${TRAVIS_BUILD_DIR} -e TRAVIS_BUILD_DIR lsstdesc/stack-sims:w_2018_09-sims_2_6_0-v2 ${TRAVIS_BUILD_DIR}/travis_test.sh
```

Comment:
`master` of this repo needs to be buildable against `master` of the LSST data management repos.  Tagging the Travis build to a specific weekly could result in confusion when DM has explicitly made something not backwards compatible.  If Travis requires that the build be tagged to a specific version, then I would ask that we not use Travis. The LSST repos have their own Jenkins cluster against which we test before merging all pull requests.  I acknowledge that system is not helpful in a fork-and-pull-request world.  We will make sure to fix anything that is broken due to a merge from a fork.

Reply:
ok thanks, I didn't know about that. Is the reason why imSim has such a travis script because it's a desc software instead of an lsst software?

Reply:
Yes.  ImSim is DESC, rather than LSST.

ImSim has also adopted policies such that it only has to build against a specified tag of the lsst_sims stack.

Good idea, though.